plugin/rewrite/README.md

```diff
@@ -209,12 +209,21 @@ rewrites the first local option with code 0xffee, setting the data to "abcd". Eq
 * A variable data is specified with a pair of curly brackets `{}`. Following are the supported variables:
   {qname}, {qtype}, {client_ip}, {client_port}, {protocol}, {server_ip}, {server_port}.
 
-Example:
+* If metadata plugin is enabled, then labels are supported as variables if they are presented within curly brackets. 
```

Comment:
If *the*. And this needs to reference the label syntax (maybe that should be a section in the metadata plugin)

Reply:
ok.